src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -337,6 +337,7 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 				if (this.isFindActive) {
 					this.addDecoration();
 				}
+				this.cellModel.cellPreviewUpdated.fire(true);
```

Comment:
If we're only ever returning true then just make the Event an `Event<void>` - the boolean doesn't provide anything useful.